save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/domain/PluginType.kt

```diff
@@ -0,0 +1,33 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.domain
+
+import com.saveourtool.save.core.config.TestConfigSections
+
+val contestAllowedPlugins = listOf(TestConfigSections.WARN)
+
+// todo: Probably should fix name in save-cli
+typealias PluginType = TestConfigSections
+
+/**
+ * @return [PluginType] from [String]
+ */
+fun String.toPluginType(): PluginType = when (this) {
+    "WarnPlugin" -> PluginType.WARN
```

Comment:
are we able to use here `com.saveourtool.save.plugin.warn.WarnPlugin::class.simpleName` and etc?

Reply:
No, because it is common section. If we add all those plugins as dependencies to common part, it will be added to frontend as well, so that the size of frontend will be increased. To my mind the game is not worth the candle.